Nodejs/Product/Nodejs/Debugger/NodeDebugger.cs

```diff
@@ -465,14 +465,10 @@ public NodeDebugger(Uri debuggerEndpointUri, int id)
                 return false;
             }
 
-            // Make step into if required
-            if (changeLiveCommand.NeedStepIn) {
+            // Make step into and update stacktrace if required
+            if (changeLiveCommand.StackModified) {
```

Comment:
I know you stated in the comments...

> It looks like it always returned the same value as "stack_modified" anyways

... but it still makes me nervous just changing the property in the condition but falling through to the same code. Has this change been verified with older version of Node?